functions/Invoke-DbaWhoisActive.ps1

```diff
@@ -254,7 +254,7 @@ Similar to running sp_WhoIsActive @get_outer_command = 1, @find_block_leaders =
         foreach ($instance in $sqlinstance) {
             try {
                 Write-Message -Level Verbose -Message "Connecting to $instance"
-                $server = Connect-SqlInstance -SqlInstance $instance -SqlCredential $sqlcredential
+                $server = Connect-SqlInstance -SqlInstance $instance -SqlCredential $sqlcredential -MinimumVersion 9
```

Comment:
This file has additional changes :/ I think you missed a sync

Reply:
Thats odd - that didnt come form me

![image](https://user-images.githubusercontent.com/6729780/35874757-42bb87f2-0b65-11e8-850b-390806f1545b.png)

Came from 926c7c253d1112f76fef0a1373a1baf777479926

How do I resolve that?